lib/engines/content_block_manager/app/components/content_block_manager/content_block/document/show/document_timeline/timeline_item_component.rb

```diff
@@ -43,4 +43,12 @@ def internal_change_note
   def change_note
     version.item.change_note
   end
+
+  def embedded_object_diffs
+    schema.subschemas.map { |subschema|
+      version.field_diffs.dig("details", subschema.id).map do |object_id, field_diff|
```

Comment:
seemed to fail locally for me if `field_diffs` was `nil`

Reply:
I've fixed this so `field_diffs` returns an empty hash by default